### PR TITLE
Update node from 8.9.1 to 8.9.2

### PR DESCRIPTION
**Pull request by SimenB**
https://nodejs.org/en/blog/release/v8.9.2/
https://github.com/nodejs/docker-node/pull/593

### CODE REVIEW

File: library/node [hunk 1]
```diff
@@ -95,31 +95,31 @@ Directory: 6/wheezy
 
 Tags: 4.8.6, 4.8, 4, argon
 Architectures: amd64, ppc64le, arm64v8, arm32v7, i386
-GitCommit: bf84a38aeacb4f6aad34e07c79fd3a0084da5cd2
+GitCommit: f941101d4d706cea27fab3b628c19e870a78d194
```
Comment (PeterDaveHello) [inline library/node]: I'm not sure if we need this change for v4?

Comment (SimenB) [reply library/node]: that was the output of the script

Comment (PeterDaveHello) [reply library/node]: Yeah I can verify that on my computer :)

**Comment by tianon** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index d838088..618a763 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -46,12 +46,12 @@ node:8.9-onbuild
 node:8.9-slim
 node:8.9-stretch
 node:8.9-wheezy
-node:8.9.1
-node:8.9.1-alpine
-node:8.9.1-onbuild
-node:8.9.1-slim
-node:8.9.1-stretch
-node:8.9.1-wheezy
+node:8.9.2
+node:8.9.2-alpine
+node:8.9.2-onbuild
+node:8.9.2-slim
+node:8.9.2-stretch
+node:8.9.2-wheezy
 node:9
 node:9-alpine
 node:9-onbuild
diff --git a/node_carbon-alpine/Dockerfile b/node_carbon-alpine/Dockerfile
index 2be0516..e9761e5 100644
--- a/node_carbon-alpine/Dockerfile
+++ b/node_carbon-alpine/Dockerfile
@@ -1,6 +1,6 @@
 FROM alpine:3.6
 
-ENV NODE_VERSION 8.9.1
+ENV NODE_VERSION 8.9.2
 
 RUN addgroup -g 1000 node \
     && adduser -u 1000 -G node -s /bin/sh -D node \
diff --git a/node_carbon-onbuild/Dockerfile b/node_carbon-onbuild/Dockerfile
index 5ad76d5..1264986 100644
--- a/node_carbon-onbuild/Dockerfile
+++ b/node_carbon-onbuild/Dockerfile
@@ -1,4 +1,4 @@
-FROM node:8.9.1
+FROM node:8.9.2
 
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
diff --git a/node_carbon-slim/Dockerfile b/node_carbon-slim/Dockerfile
index b2ae317..8221241 100644
--- a/node_carbon-slim/Dockerfile
+++ b/node_carbon-slim/Dockerfile
@@ -20,7 +20,7 @@ RUN set -ex \
     gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$key" ; \
   done
 
-ENV NODE_VERSION 8.9.1
+ENV NODE_VERSION 8.9.2
 
 RUN buildDeps='xz-utils' \
     && ARCH= && dpkgArch="$(dpkg --print-architecture)" \
diff --git a/node_carbon-stretch/Dockerfile b/node_carbon-stretch/Dockerfile
index ad3aa3e..3682bef 100644
--- a/node_carbon-stretch/Dockerfile
+++ b/node_carbon-stretch/Dockerfile
@@ -20,7 +20,7 @@ RUN set -ex \
     gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$key" ; \
   done
 
-ENV NODE_VERSION 8.9.1
+ENV NODE_VERSION 8.9.2
 
 RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
   && case "${dpkgArch##*-}" in \
diff --git a/node_carbon-wheezy/Dockerfile b/node_carbon-wheezy/Dockerfile
index 3013293..0557615 100644
--- a/node_carbon-wheezy/Dockerfile
+++ b/node_carbon-wheezy/Dockerfile
@@ -20,7 +20,7 @@ RUN set -ex \
     gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$key" ; \
   done
 
-ENV NODE_VERSION 8.9.1
+ENV NODE_VERSION 8.9.2
 
 RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
   && case "${dpkgArch##*-}" in \
diff --git a/node_carbon/Dockerfile b/node_carbon/Dockerfile
index 7eaff73..81b7926 100644
--- a/node_carbon/Dockerfile
+++ b/node_carbon/Dockerfile
@@ -20,7 +20,7 @@ RUN set -ex \
     gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$key" ; \
   done
 
-ENV NODE_VERSION 8.9.1
+ENV NODE_VERSION 8.9.2
 
 RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
   && case "${dpkgArch##*-}" in \
````

</details>

**Comment by tianon** [comment]
LGTM

Build test of #3781; 54d38ad0; `amd64` (`node`):

```console

$ bashbrew build node:9.2.0
Using bashbrew/cache:6412bfc144f44cd9499407eb8e9c57ef3ed9212a4790a7442fc5b38a112f6207 (node:9.2.0)
Tagging node:9.2.0
Tagging node:9.2
Tagging node:9
Tagging node:latest

$ test/run.sh node:9.2.0
testing node:9.2.0
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:9.2.0-alpine
Using bashbrew/cache:81239775b69388741bcd4bd97e00824318d89b366988cbae88e789d341c74122 (node:9.2.0-alpine)
Tagging node:9.2.0-alpine
Tagging node:9.2-alpine
Tagging node:9-alpine
Tagging node:alpine

$ test/run.sh node:9.2.0-alpine
testing node:9.2.0-alpine
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:9.2.0-onbuild
Using bashbrew/cache:62ac7b1296fc944dc61d9129c5a1dbbffb7baae472a12407e4d8da919c37dde5 (node:9.2.0-onbuild)
Tagging node:9.2.0-onbuild
Tagging node:9.2-onbuild
Tagging node:9-onbuild
Tagging node:onbuild

$ test/run.sh node:9.2.0-onbuild
testing node:9.2.0-onbuild
	'override-cmd' [1/1]...passed


$ bashbrew build node:9.2.0-slim
Using bashbrew/cache:953b48d5b84f4fbcd4225c2774fbe6e07adad87c2d69270410728628d9352991 (node:9.2.0-slim)
Tagging node:9.2.0-slim
Tagging node:9.2-slim
Tagging node:9-slim
Tagging node:slim

$ test/run.sh node:9.2.0-slim
testing node:9.2.0-slim
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:9.2.0-stretch
Using bashbrew/cache:0ed59137a1c10d11832d379a49bd3b9e6a65a0fe27ffdd5f725278221dd7173f (node:9.2.0-stretch)
Tagging node:9.2.0-stretch
Tagging node:9.2-stretch
Tagging node:9-stretch
Tagging node:stretch

$ test/run.sh node:9.2.0-stretch
testing node:9.2.0-stretch
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:9.2.0-wheezy
Using bashbrew/cache:c6c7578bebeb14e103503f679f86bd2d5789706f405af7d0186c141d4161990f (node:9.2.0-wheezy)
Tagging node:9.2.0-wheezy
Tagging node:9.2-wheezy
Tagging node:9-wheezy
Tagging node:wheezy

$ test/run.sh node:9.2.0-wheezy
testing node:9.2.0-wheezy
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:8.9.2
Building bashbrew/cache:59e356157aa677d971bfc090a9cb3a5120e767b3333cc6716e056714578c9d2d (node:8.9.2)
Tagging node:8.9.2
Tagging node:8.9
Tagging node:8
Tagging node:carbon

$ test/run.sh node:8.9.2
testing node:8.9.2
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:8.9.2-alpine
Building bashbrew/cache:5509fece2e110e23cddd4fc06c71f6f76e568426d1ae5762155feafa89585bdf (node:8.9.2-alpine)
Tagging node:8.9.2-alpine
Tagging node:8.9-alpine
Tagging node:8-alpine
Tagging node:carbon-alpine

$ test/run.sh node:8.9.2-alpine
testing node:8.9.2-alpine
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:8.9.2-onbuild
Building bashbrew/cache:4417bb8e7b021632ddbf2bd91838df8770a032767ad14e4b7cae56ee88370fac (node:8.9.2-onbuild)
Tagging node:8.9.2-onbuild
Tagging node:8.9-onbuild
Tagging node:8-onbuild
Tagging node:carbon-onbuild

$ test/run.sh node:8.9.2-onbuild
testing node:8.9.2-onbuild
	'override-cmd' [1/1]...passed


$ bashbrew build node:8.9.2-slim
Building bashbrew/cache:f639297cc2c3161913c2710bf043d4272cc48c875d576ee9be92b4b24e6eecd5 (node:8.9.2-slim)
Tagging node:8.9.2-slim
Tagging node:8.9-slim
Tagging node:8-slim
Tagging node:carbon-slim

$ test/run.sh node:8.9.2-slim
testing node:8.9.2-slim
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:8.9.2-stretch
Building bashbrew/cache:2a42df4cbf79166c15b91cc91ec871126a699c158075e8f0884be667769db11a (node:8.9.2-stretch)
Tagging node:8.9.2-stretch
Tagging node:8.9-stretch
Tagging node:8-stretch
Tagging node:carbon-stretch

$ test/run.sh node:8.9.2-stretch
testing node:8.9.2-stretch
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:8.9.2-wheezy
Building bashbrew/cache:01aa623d51298795e56200459d1bb8e6f25d44aa4556b2b8c3df3ff64992a1aa (node:8.9.2-wheezy)
Tagging node:8.9.2-wheezy
Tagging node:8.9-wheezy
Tagging node:8-wheezy
Tagging node:carbon-wheezy

$ test/run.sh node:8.9.2-wheezy
testing node:8.9.2-wheezy
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:6.12.1
Using bashbrew/cache:a1c097f059f129f776cd377d5e3d849049640fe90905856d96d605ebfe574163 (node:6.12.1)
Tagging node:6.12.1
Tagging node:6.12
Tagging node:6
Tagging node:boron

$ test/run.sh node:6.12.1
testing node:6.12.1
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:6.12.1-alpine
Using bashbrew/cache:5e148162b5a18e8efbc3330899304bff1da8d633860a497e7c2068c685a5a168 (node:6.12.1-alpine)
Tagging node:6.12.1-alpine
Tagging node:6.12-alpine
Tagging node:6-alpine
Tagging node:boron-alpine

$ test/run.sh node:6.12.1-alpine
testing node:6.12.1-alpine
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:6.12.1-onbuild
Using bashbrew/cache:e20e7128fa94eb2e388da161a4c86a19a6593b840e2356cb5e83a168cabdee3f (node:6.12.1-onbuild)
Tagging node:6.12.1-onbuild
Tagging node:6.12-onbuild
Tagging node:6-onbuild
Tagging node:boron-onbuild

$ test/run.sh node:6.12.1-onbuild
testing node:6.12.1-onbuild
	'override-cmd' [1/1]...passed


$ bashbrew build node:6.12.1-slim
Using bashbrew/cache:c6f8d467d7568c37d899b2beff11f465d3b11612b3a613f8fadefbe42f972e0f (node:6.12.1-slim)
Tagging node:6.12.1-slim
Tagging node:6.12-slim
Tagging node:6-slim
Tagging node:boron-slim

$ test/run.sh node:6.12.1-slim
testing node:6.12.1-slim
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:6.12.1-stretch
Using bashbrew/cache:ade4a25fe1721c0679bc59ba3f397cfddfa34d5d29ddbabea49cf91a6f441c8d (node:6.12.1-stretch)
Tagging node:6.12.1-stretch
Tagging node:6.12-stretch
Tagging node:6-stretch
Tagging node:boron-stretch

$ test/run.sh node:6.12.1-stretch
testing node:6.12.1-stretch
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:6.12.1-wheezy
Using bashbrew/cache:56f12f7318690b54365772c1b05152fec608a794090f8e5fd16b212c8c59622f (node:6.12.1-wheezy)
Tagging node:6.12.1-wheezy
Tagging node:6.12-wheezy
Tagging node:6-wheezy
Tagging node:boron-wheezy

$ test/run.sh node:6.12.1-wheezy
testing node:6.12.1-wheezy
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:4.8.6
Building bashbrew/cache:c84522a724bfb7eb09fb45650a328074ebd4f30d03f76376f995baafccee6413 (node:4.8.6)
Tagging node:4.8.6
Tagging node:4.8
Tagging node:4
Tagging node:argon

$ test/run.sh node:4.8.6
testing node:4.8.6
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:4.8.6-alpine
Building bashbrew/cache:22e95709773adbfbc9400d83114beab2e8ee1eb44eaf82d73afe0e470ef5a64f (node:4.8.6-alpine)
Tagging node:4.8.6-alpine
Tagging node:4.8-alpine
Tagging node:4-alpine
Tagging node:argon-alpine

$ test/run.sh node:4.8.6-alpine
testing node:4.8.6-alpine
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:4.8.6-onbuild
Building bashbrew/cache:70c55711c3cfca7268c5b7e2036e7d8c050cebf688f004da702e8490cdbe3d1e (node:4.8.6-onbuild)
Tagging node:4.8.6-onbuild
Tagging node:4.8-onbuild
Tagging node:4-onbuild
Tagging node:argon-onbuild

$ test/run.sh node:4.8.6-onbuild
testing node:4.8.6-onbuild
	'override-cmd' [1/1]...passed


$ bashbrew build node:4.8.6-slim
Building bashbrew/cache:c065672a16596dbbb26e588456d40de9a01b680659a6710d08348ba228c99cab (node:4.8.6-slim)
Tagging node:4.8.6-slim
Tagging node:4.8-slim
Tagging node:4-slim
Tagging node:argon-slim

$ test/run.sh node:4.8.6-slim
testing node:4.8.6-slim
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:4.8.6-stretch
Building bashbrew/cache:44bb93a02d9d00b1d489cac8c28aa79a20f6025c671b093fcec4f4343615f191 (node:4.8.6-stretch)
Tagging node:4.8.6-stretch
Tagging node:4.8-stretch
Tagging node:4-stretch
Tagging node:argon-stretch

$ test/run.sh node:4.8.6-stretch
testing node:4.8.6-stretch
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed


$ bashbrew build node:4.8.6-wheezy
Building bashbrew/cache:5ec26428d04ee7bac8b4d9a96fe17f478fa636379c60a0123ce4d276a623f520 (node:4.8.6-wheezy)
Tagging node:4.8.6-wheezy
Tagging node:4.8-wheezy
Tagging node:4-wheezy
Tagging node:argon-wheezy

$ test/run.sh node:4.8.6-wheezy
testing node:4.8.6-wheezy
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'node-hello-world' [5/5]...passed

```

**Pull request merged**